hw05_parallel_execution/go.mod

```
@@ -1,16 +1,16 @@
-module github.com/fixme_my_friend/hw05_parallel_execution
+module github.com/flatmix/hw05_parallel_execution
```

```suggestion
module github.com/flatmix/hw-test/hw05_parallel_execution
```
не забывайте, пожалуйста, добавлять название репозитория при редактировании пути